tests/base/mpi_evenly_distributed_partitioning_01.output

```diff
@@ -0,0 +1,4 @@
+
+DEAL:0::Total size=17
```

Comment:
The file name needs to be `tests/base/mpi_evenly_distributed_partitioning_01.mpirun=1.output` to avoid trouble when running the tests simultaneously.